openadapt/events.py

```diff
@@ -129,7 +141,7 @@ def get_events(
         event="get_events.completed", properties={"recording_id": recording.id}
     )
 
-    return action_events  # , window_events, screenshots
+    return action_events, browser_events  # , window_events, screenshots
```

Comment:
I don't think we need to return browser_events here do we?

Reply:
Done here: 1bcc2ca8e0aba96288bda3ba14fd0a98c6e9047f